backend/controllers/orgUsers.js

```diff
@@ -9,7 +9,7 @@ const getAllOrgUsers = async (req, res) => {
 const createOrgUser = async (req, res) => {
   try {
     const orgUser = await OrgUser.create(req.body);
-    res.status(200).json(orgUser);
+    res.status(201).json({message: "Thanks! your account has been successfully created"});
   } catch (error){
     res.status(400).json({error: error.message});
   }
```

Comment:
Hey re line 4 'const getAllOrgUsers' I'm not sure why we would require get all org users, are you able to advise?

Looking at Acebook we had findUserById
`FindUserById: (req, res) => {
        // Extract user_id from params (Taken from :post_id in the url)
        let user_id = req.params.user_id
        // Use user_id to find particular post
        User.findById({_id:user_id}, async (err, user) => {
            if (err) {
                throw err;
            }
            // const token = await TokenGenerator.jsonwebtoken(req.user_id)
            // returns a body containing the post object and token string
            res.status(200).json({user});
        });
    },`

Reply:
yeah this is on the backlog we were going to create separate tickets for these.

also, the get all users is for testing purposes just so we have the route available to us in postman :)